de.bforartists.Bforartists.json

```diff
@@ -0,0 +1,179 @@
+{
+    "id": "de.bforartists.Bforartists",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "bforartists",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=home",
+        "--filesystem=/mnt",
+        "--filesystem=/media",
+        "--filesystem=/run/media",
+        "--env=TMP_DIR=/tmp",
+        "--env=TMP=/tmp"
+    ],
+    "add-extensions": {
+        "org.blender.Blender.Codecs": {
+            "directory": "lib/codecs",
+            "add-ld-path": "lib",
+            "bundle": true,
+            "autodelete": true
+        },
+        "org.freedesktop.Platform.ffmpeg-full": {
```

Comment:
why this if you build ffmpeg?

Reply:
The Blender flatpak team tried moving over to Freedesktop's FFmpeg to allow for Blender to be used with only free codecs, but found that openh264 did not work well with Blender. They then went back to bundling their own build of FFmpeg, but now as an extension to still allow for free builds of Blender to be used.

Initial move to `org.freedesktop.Platform.ffmpeg-full`: https://github.com/flathub/org.blender.Blender/pull/43

Move back to bundled as an extension: https://github.com/flathub/org.blender.Blender/pull/47

I can remove one or the other if you think that would be best for this Bforartists package, but their solution seems good to me.

Reply:
I would drop `ffmpeg-full` extension. You already bundle your own, the one from extension is not going to be used.